judger/src/tester/exec.rs

```diff
@@ -558,7 +581,15 @@ impl TestSuite {
             None
         };
 
+        let network_name =
```

Comment:
You can `then` if you like, although it's no big deal here ;)

Reply:
Fixed, because it's not needed.